template/lab/samples/ProjectTemplate.Samples/MultiTarget.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+    <PropertyGroup>
+        <!--
+          MultiTarget is a custom property that indicates which target a project is designed to be built for / run on.
+          Used to create project references, generate solution files, enable/disable TargetFrameworks, and build nuget packages.
+        -->
+        <MultiTarget>uwp;wasdk;wpf;wasm;linuxgtk;macos;ios;android;</MultiTarget>
```

Comment:
Shouldn't the samples just match the source configuration? Do we need to have two copies of this?

Reply:
We do for the MVP at least. Adding this will take additional time and won't be a breaking change down the road since it's included in all templates. We'll need to add minimal docs for how this works and how to use it.

 The MultiTarget system (the files in `./common/MultiTarget/`) is generating per-project and doing strictly MultiTarget things. To keep things clean and maintainable, we'll want to build extended functionality on top of what we have, rather than into it.

Reply:
I'll spin something up

Reply:
Done, fixed in 14f52458574bc667be94312febca6614230cd01f